lib/money_column/active_record_hooks.rb

```diff
@@ -7,25 +18,7 @@ def self.included(base)
     module ClassMethods
       def money_column(*columns)
```

Comment:
Should we deprecate this method? Not sure if it adds much value over using the attribute API directly.